website/sidebars.json

```diff
@@ -10,32 +10,6 @@
       "en/contributing/policies"
     ],
     "Packages": [
-      "en/packages/fast-animation/index",
```

Comment:
Are these all now being auto-generated?  Should this PR include the result of running the new build scripts?

Reply:
I wasn't sure if I should include the outputted result in the PR, if it was an empty array I thought it'd be easier to see how everything in the array would be regenerated when the script is run.

Reply:
Sure, that makes sense.